subiquity/controllers/filesystem.py

```diff
@@ -255,7 +254,7 @@ def start_ui(self):
             # performed would be tricky.  Possibly worth doing though! Just
             # not today.
```

Comment:
Can we take the opportunity to make this go; either fix the issue now as a refactor or file a bug to handle later? Is it even relevant with the current rework of the UI?